log-manager/src/main/java/io/airlift/log/LoggingConfiguration.java

```diff
@@ -104,4 +105,15 @@ public LoggingConfiguration setLevelsFile(String levelsFile)
         this.levelsFile = levelsFile;
         return this;
     }
+
+    public boolean useJson()
+    {
+        return useJson;
+    }
+
+    @Config("log.json")
```

Comment:
I would model this as a setting that controls the logging format, instead, with possible values "TEXT" and "JSON"